internal/storagenode/logstream/replicate_client_test.go

```diff
@@ -123,7 +123,7 @@ func TestReplicateClientRPCError(t *testing.T) {
 	rc.queue = make(chan *replicateTask, 1)
 	rc.streamClient = mockStreamClient
 
-	rt := newReplicateTask(0)
+	rt := newReplicateTaskDeprecated(0)
```

Comment:
It seems that it can be replaced with `newReplicateTask(1)`. Is there a reason to use a function that is scheduled to be deprecated?

Reply:
It was removed in the next PR.